docs/modules/ROOT/pages/data-migration.adoc

```diff
@@ -0,0 +1,53 @@
+= Migrate Data to another cluster
+
+The goal of this document is to describe the way how to migrate data from one cluster to another.
+
+== Create an Enterprise Cluster
+
+To learn how to create a cluster, see xref:create-enterprise-cluster.adoc[].
+
+Highly recommended creating a cluster with the same configuration, so if your existed cluster
+contains xref:custom-classes-upload.adoc[custom classes], xref:backup-and-restore.adoc[backup] or xref:persistence.adoc[persistence],
+it makes sense to enable/configure these options for a new cluster as well.
+
+== Create WAN Replication
+
+In this tutorial, we are going to use a fantastic Hazelcast feature for data migration which is called xref:wan-replication.adoc[WAN Replication]
+
+In order to create WAN Replication, you can go to *Cluster Detail* > *Settings* > *Wan Replication* and click *Create New Replication*
+
+image:aws-cluster-settings.png[Go to Manage Cluster > Settings]
+
+image:wan-replication-settings.png[WAN Replication settings]
+
+image:configure-wan-replication.png[WAN Replication configuration]
+
+After setting all these parameters, you can click *Next* to select and configure consumer cluster.
+
+image:configure-wan-replication-2.png[Part 2 of the WAN Replication configuration]
+
+Settings on this screen for consumer cluster are same as publisher cluster except *Active/Passive* section. If you set consumer cluster as *Active*, data will be synchronized from/to publisher and consumer. If you set consumer cluster as *Passive*, data will be sent only from publisher to consumer.
+Once you configured settings for consumer, you can click *Save*
+
+image:wan-replication-in-progress.png[Wan Replication is being set up]
+
+Within seconds, wan replication will be ready there.
+
+image:wan-replication-successful.png[Wan Replication is active]
+
+Once it is finished, you can verify it by going to management center *Cluster Detail* > *Management Center* > *Enable*  then *Management Center* > *Open Management Center*.
+
+image:wan-replication-mc.png[Wan Replication data in Management Center]
+
+When you connect publish cluster and try to write some data, you will see data will be replicated to consumer cluster.
+
+NOTE: You can synchronize all data by using `Sync` button in the WAN replication settings.
+
+== Connect to a Cluster
+
+After migration is done, you can connect to a new cluster via different clients, e.g. xref:java-client.adoc[]
+
+
+== Delete a Cluster
```

Comment:
I think they need to delete WAN replication first?

Reply:
No, they should completely migrate all clients first and only after a while delete wan and cluster itself.

Reply:
Correct, but there is nothing about - "delete WAN". So I think it worth mentioning that WAN needs to be deleted and after that, you can delete the old cluster.

Reply:
done